src/webui/www/private/views/statistics.html

```diff
@@ -1,3 +1,4 @@
+<div id="statisticsContent">
```

Comment:
Can you indent the content of this `div`.

Reply:
@Piccirello  Fixed, and also removed some extra newlines. This way it is more inline with the formatting of other files such as `about.html`

Reply:
>Fixed, and also removed some extra newlines. 

Not sure about others, I actually find having an empty line between `</table>` and `<h3>` is more readable.